Docs: AWS IC System Credentials Not Supported on Cloud Clusters (#66102)
gravitational/teleport: docs/pages/identity-governance/integrations/aws-iam-identity-center/migrating-identity-center-from-okta-to-teleport.mdx

Again, we need to install the plugin using `tctl`.

Choose the installation command for your Teleport deployment:

<Tabs>
<TabItem label="Teleport Cloud">

> 

Reviewer: Same as above.

Maybe even convert this into a partial/include? So it can be reused between different pages.

Author: I tried that a bit but soon realized the content under "Teleport Cloud" tab isn't exactly the same across pages, mainly due to different arguments being used in the example `tctl` commands as a result of a broader context difference (okta migration v.s. pure awsic setup). So I opted not to make either tab portable.

Using system-level credential is also the recommended way to provide AWS credential when
configuring integration in the Teleport Enterprise self-hosted deployment.

> Teleport Cloud clusters cannot use `--use-system-credentials`, though. If you

Reviewer 2: ```suggestion
Teleport Cloud clusters cannot use `--use-system-credentials`. If you
```

> are on Teleport Cloud, please set up the

Reviewer 2: ```suggestion
On Teleport Cloud, use the [AWS OIDC integration](../../../enroll-resources/application-access/cloud-apis/awsoidc-integration.mdx) instead.
```

> [AWS OIDC integration](../../../enroll-resources/application-access/cloud-apis/awsoidc-integration.mdx)

Reviewer 2: ```suggestion
```

> and use that instead.

Reviewer 2: ```suggestion
```

Reviewer 2: Making this sentence a little tighter; per our style guide, avoid using "please" in instructions.

Using system-level credential is also the recommended way to provide AWS credential when
configuring integration in the Teleport Enterprise self-hosted deployment.

> Teleport Cloud clusters cannot use `--use-system-credentials`. On Teleport Cloud, use the

Reviewer: Same comment as above.
